.changelog/1.0.1.toml

```diff
@@ -0,0 +1,8 @@
+release-date = "2023-04-03"
```

Comment:
```suggestion
```

Same here, and that would go into `.changelog/_unreleased.toml` until I run `slap release` after merge.